daemon/src/engine/sys/sdl_glimp.cpp

```diff
@@ -769,7 +769,7 @@ static int GLimp_SetMode( int mode, bool fullscreen, bool noborder )
 				continue;
 			}
 			SDL_GL_SetSwapInterval( r_swapInterval->integer );
-			SDL_ShowCursor( 0 );
+			//SDL_ShowCursor( 0 );
```

Comment:
Any reason to leave it commented? If its not necessary, might be best to just delete it.